package.json

```
@@ -4,10 +4,11 @@
   "private": true,
   "license": "Apache-2.0",
   "devDependencies": {
-    "@aws-sdk/config-resolver": "^3.4.1",
-    "@aws-sdk/middleware-stack": "^3.3.0",
-    "@aws-sdk/node-config-provider": "^3.4.1",
-    "@aws-sdk/smithy-client": "^3.4.1",
+    "@aws-sdk/config-resolver": "^3.18.0",
+    "@aws-sdk/middleware-stack": "^3.18.0",
+    "@aws-sdk/node-config-provider": "^3.18.0",
+    "@aws-sdk/smithy-client": "^3.18.0",
+    "@aws-sdk/types": "^3.18.0",
```

Did you mean `^3.0.0` to maximize compatibility?

```suggestion
    "@aws-sdk/types": "^3.0.0",
```